docs/content/integrations/embedded-elt/dlt.mdx

```diff
@@ -116,21 +164,25 @@ DESTINATION__SNOWFLAKE__CREDENTIALS__WAREHOUSE=""
 DESTINATION__SNOWFLAKE__CREDENTIALS__ROLE=""
 ```
 
-Ensure that these are defined in your environment.
+Ensure that these variables are defined in your environment.
+
+---
 
-## 4. Define a `DagsterDltResource`
+## Step 4: Define a DagsterDltResource
 
-The <PyObject module="dagster_embedded_elt.dlt" object="DagsterDltResource" /> provides a wrapper of a _dlt_ pipeline runner. This can be shared across _dlt_ pipelines, and is a required parameter to our _dlt_ asset definition.
+The <PyObject module="dagster_embedded_elt.dlt" object="DagsterDltResource" /> provides a wrapper of a dlt pipeline runner. This can be shared across dlt pipelines and is a required parameter to our dlt asset definition.
 
 ```python
 from dagster_embedded_elt.dlt import DagsterDltResource
 
 dlt_resource = DagsterDltResource()
 ```
 
-## 5. Create a `dlt_assets` definition for GitHub
+---
+
+## Step 5: Create a dlt_assets definition for GitHub
 
-The `dlt_assets` decorator takes a `dlt_source` and `dlt_pipeline` parameter. In this example, we will use the `github_reactions` source and create a `dlt_pipeline` to ingest from Github to Snowflake.
+The `dlt_assets` decorator takes a `dlt_source` and `dlt_pipeline` parameter. In this example, we used the `github_reactions` source and created a `dlt_pipeline` to ingest data from Github to Snowflake:
```

Comment:
What file should users be doing this in?

Also, we should show the imports being used as well. Code snippets should be as copy-and-pasteable as possible.